doc/source/programs/gdal_raster_update.rst

```diff
@@ -0,0 +1,122 @@
+.. _gdal_raster_update:
+
+================================================================================
+``gdal raster update``
+================================================================================
+
+.. versionadded:: 3.12
+
+.. only:: html
+
+    Update the destination raster with the content of the input one.
+
+.. Index:: gdal raster update
+
+Synopsis
+--------
+
+.. program-output:: gdal raster update --help-doc
+
+Description
+-----------
+
+:program:`gdal raster update` can be used to update an existing target raster dataset
+with the pixel values of the source raster, doing reprojection if the source
+and target datasets do not have the same coordinate reference systems.
+The extent, size, resolution or coordinate reference system of the target dataset
+are not modified by this operation.
+
+Overviews are *not* updated by this command, but may be if using :ref:`gdal_raster_overview_refresh`.
+
+Standard options
+++++++++++++++++
+
+.. option:: -r, --resampling <RESAMPLING>
```

Comment:
Worth pulling this out into a separate file that can be included?

Reply:
done